components/04-views/dashboard/gift-registry/gift-registry--form.hbs

```diff
@@ -0,0 +1,31 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            {{> '@dashboard-form' generalInformation }}
+
+            {{> '@dashboard-form' eventInformation }}
+
+            {{> '@dashboard-form' eventInformationSecondPart }}
+
+            {{> '@dashboard-form' registrantInformation }}
+
+            {{> '@dashboard-actions' addRegistrant }}
+
+            {{> '@dashboard-form' shippingAddress }}
+
+            {{> '@dashboard-form' hiddenNewAddressForm }}
+
+            {{> '@dashboard-actions' buttons }}
+        </div>
+    </div>
+</div>
+
+{{ render '@footer' }}
+
+<script src="{{ static '../gift-registry/gift-registry--form.js' }}"></script>
```

Comment:
all forms fields (for all dashboards with content in one column) are 100% width for desktop view. Is it ok? Wouldn't be better to make it some max-width for wider screens?

Reply:
Actually, I have an issue in the past with this and I would rather leave this in 100% width b/c giving into form `max-width` IMO is some layout thing. And I remember to using some form or inputs in Janus theme and have a problem with overwriting this `max-width`. Then I must edit component which I created and push it here without this. But from the other side it's dashboard view, and in most projects, we don't have here anything. And don't know what to do with this. But the first thought when I saw this issue was 'Leave it full width'

Reply:
ok, so leave it full wide, if we will done some styling for specific design, we will be working on it

Reply:
👍